app/controllers/searches_controller.rb

```diff
@@ -213,7 +213,7 @@ def inactive_friends
   def inactive_followers
     @user_items = build_user_items(@searched_tw_user.inactive_followers)
     @graph = @searched_tw_user.inactive_followers_graph
-    @tweet_text = view_context.inactive_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user)
+    @tweet_text = view_context.inactive_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user)
```

Comment:
__[rubocop]__
* Line is too long. [91/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)